monorepo-builder.yml

```diff
@@ -70,7 +70,7 @@ parameters:
     src/Gitea: 'git@github.com:SocialiteProviders/Gitea.git'
     src/Gitee: 'git@github.com:SocialiteProviders/Gitee.git'
     src/Goodreads: 'git@github.com:SocialiteProviders/Goodreads.git'
-    src/Google: 'git@github.com:SocialiteProviders/Google-Plus.git'
+    src/Google: 'git@github.com:SocialiteProviders/Google.git'
```

Comment:
Note: need to rename this repo

Reply:
do you mean remove? Google+ has become redundant, no?